### ISSUE TITLE
PE loader should check for gnu3 style demangling in addition to msvc

### ISSUE THREAD

**Issue by psifertex** [Vector35/binaryninja-api#2405]
Some 

**To Reproduce**
1. Open a PE file compiled via Clang with C++ names and don't strip. 

**Expected behavior**
Mangled names should be demangled easier to read names.

**Comment by psifertex** [comment]
Marking as low impact only because there's a very easy work-around for impacted binaries:

```
for fn in bv.functions:
  if fn.name.startswith("_Z"):
    fn.name = demangle_gnu3(Architecture['x86_64'], fn.name)
 ```

**Comment by cxnder** [comment]
Applicable binary: https://mirror.msys2.org/mingw/mingw64/mingw-w64-x86_64-atkmm-2.36-2.36.2-1-any.pkg.tar.zst

**Comment by cxnder** [comment]
Resolved in >= 3.1.3593
